uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html

```diff
@@ -23,7 +23,7 @@ <h4 class="modal-title">New Document</h4>
         </div>
       </div>
       <!-- Document Type -->
-      <div class="row" ng-hide="isRole('staff')">
+      <div class="row" ng-hide="isRole('staff')" ng-show="!isRole('manager')">
```

Comment:
Do not include `ng-hide`and `ng-show` in the same markup, you can just add your new condition in `ng-hide`with `||` or `&&` as applicable